.circleci/config.yml

```diff
@@ -106,13 +111,33 @@ jobs:
               --fork-url="https://ci-sepolia-l1.optimism.io" --compute-units-per-second=320
             forge script CheckSecurityConfigs \
               --fork-url="https://ci-goerli-l1.optimism.io" --compute-units-per-second=320
+      - notify-failures-on-main
 
 workflows:
+  nightly:
+    jobs:
+      - golang-test:
+          context:
+            - slack
+      - check-security-configs:
+          context:
+            - slack
+    triggers:
+      - schedule: 
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - main
```

Comment:
In the monorepo we do this by defining [triggers](https://app.circleci.com/settings/project/github/ethereum-optimism/optimism/triggers) and using them [like this](https://github.com/ethereum-optimism/optimism/blob/39bc6f433c3344079dc4933688ad2e72064aec99/.circleci/config.yml#L2025-L2031).

I'm not sure if there's any reason to prefer that over the cron approach though.

Reply:
Yes I had spotted that different approach. I believe circle tried to [deprecate scheduled workflows in favour of scheduled pipelines, but decided not ](https://circleci.com/docs/migrate-scheduled-workflows-to-scheduled-pipelines/)to in the end (yet). 

I don't have a very strong opinion either way, other than that this way is already done. Another thing I like is that all the config is in one place next to the source code and I don't need to go hunting in some other dashboard to understand what is going to happen.

Reply:
Cool I'm onboard with leaving this config as-is, will resolve this comment